YHP_website/src/main/webapp/WEB-INF/pages/account.jsp

```diff
@@ -57,43 +57,68 @@
 						{
 							alert(response.error);
 						}
-						else
-						{
-							alert("success");
-						}
 					}
 				}
 				xhttp.send('user=' + JSON.stringify(formData));
 			}
+		
 		</script>
 	</head>
 	<body>
 		<div class="container-fluid">
 			<%@ include file="./includes/header.jspf" %>
+			<%!
+			private String getUserAttribute(boolean isEditing, String attributeName, String attributeValue)
+			{
+				final String formType = isEditing? "":" text";
+				final String readOnly = isEditing? "":" readonly='readonly'";
+				
+				//Ternary operator didn't seem to like attributeValue == null ? attributeValue : "";
+				if (null == attributeValue)
+				{
+					attributeValue = ""; 
+				}
+				
+				return String.format("<label>%s: <input value='%s' type='text' class='form-control%s' name='%s' title='%s'%s", 

```

Comment:
need `type='password'` for password fields. potentialy give this method a optional parameter for type that defaults to 'text'